player/client.c

```diff
@@ -1035,10 +1035,12 @@ static void cmd_complete(struct mp_cmd_ctx *cmd)
 
 static int run_client_command(mpv_handle *ctx, struct mp_cmd *cmd, mpv_node *res)
 {
-    if (!ctx->mpctx->initialized)
-        return MPV_ERROR_UNINITIALIZED;
     if (!cmd)
         return MPV_ERROR_INVALID_PARAMETER;
+    if (!ctx->mpctx->initialized) {
+        mp_cmd_free(cmd);
```

Comment:
should probably call `talloc_free` for consistency (no other code uses mp_cmd_free)